charts/kong-collectorapi/values.yaml

```diff
@@ -1,6 +1,6 @@
 image:
   repository: kong-docker-kong-brain-immunity-base.bintray.io/kong-brain-immunity
-  tag: 3.0.0
+  tag: 4.0.0
   pullPolicy: IfNotPresent
 
 imagePullSecrets:
```

Comment:
If I understand correctly immunity doesn't need to contact kong so the whole section about kongAdmin can go and anywhere its referenced

Reply:
nevermind. we still use it for looking up things